src/pages/Account/Venue/VenueMapEditor/DraggableSubvenue.module.scss

```diff
@@ -0,0 +1,22 @@
+.resizeableImageContainer {
+  overflow: hidden;
+  width: 100%;
+  flex: 1;
+  background-position: center;
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+.resizeTab {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  background-color: white;
+  border: 1px solid var(--content--over);
```

Comment:
These styles look like just copied over from before. The `var(--content--over)` will most likely not be there and make border not show up. Similar comment for the rest of the hard-coded values.